src/App.css

```diff
@@ -47,3 +47,9 @@ input.error {
   box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.075),
     0 0 0 3px rgba(255, 0, 0, 0.1);
 }
+select.error {
+  border-color: red;
```

Comment:
you are repeating logic from lines 45 to 49 here,

these lines can be combined.

```css
input.error,
select.error {
  border-color: red;
  box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.075), 0 0 0 3px rgba(255, 0, 0, 0.1);
}
```

Reply:
combined css logic in one